packages/react-renderer-demo/src/pages/migration-guide-v3.md

```diff
@@ -0,0 +1,73 @@
+import DocPage from '@docs/doc-page';
+
+<DocPage>
+
+# Migration guide to version 3
+
+This new version does not bring any new features, but is aimed to improve developer experience by integrating a new build process to reduce bundle sizes and to introduce automatic ESM/CJS module resolution.
+
+---
+
+## Removal of PatternFly 3 Mapper
+
+PatternFly 3 mapper is now removed. You can still use the mapper from version 2, but we can't guarantee any future bug fixes or new features. However, we are able to assist you with your PRs and we can still release them as you need.
+
+**We strongly recommend to migrate to some other mapper** (mainly to PF4), as PF3 package is no longer maintained well and it will have issues with the latest version of React.
+
+Migration should be simple - most of the components share the same API so replacing imports should work fine in most cases.
+
+---
+
+## New build process
+
+### Automatic module resolution
+
+You don't have to specify the correct module, your environment will do it automatically, if corectly configured. See [optimization page](/optimization)  for more details.
+
+### No UMD packages
+
+UMD format is no longer supported. The import paths for UMD packages now lead to CJS modules in testing environments and ESM modules in browser environments.
+
+### Changed import paths
+
+Due to the new build process, you have to change your imports path, if you use relative imports of specific components or if you consume the specific module system package. There are no longer module system and the dist folder specified.
+
+**Before**
+
+`import useFieldApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-field-api';`
```

Comment:
I think we could use the git diff syntax to better illustrate these changes:
```diff
-import useFieldApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-field-api';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';`
```
I am not sure though if the current code component supports it

Reply:
Yeah.. I didn't use the editor because there is the automatic import paths changer. 😅 

But diff is supported, so I will reimplement the ability to turn off the import switching.